### ISSUE TITLE
Wrapping geocodePostalAddress into Promise

### ISSUE THREAD

**Issue by timbms** [PromiseKit/CoreLocation#11]
I want to propose an addition to wrap geocodePostalAddress available as of iOS 11 into PromiseKit
```swift 
    /// Submits a forward-geocoding request using the specified postal address.
    @available(iOS 11.0, *)
    public func geocodePostalAddress(_ postalAddress: CNPostalAddress) -> Promise<[CLPlacemark]> {
        return Promise { seal in
            geocodePostalAddress(postalAddress, completionHandler: seal.resolve)
        }
    }

```

**Comment by mxcl** [comment]
k, you want to PR? Or I can probably do it today.

**Comment by timbms** [comment]
You can do it. I haven't done PR on GitHub so far.

**Comment by mxcl** [comment]
I make no promises, I have very little time for free open source work today.

**Comment by nathanhosselton** [comment]
@timbms A small, purely additive change like this is the perfect way to get your feet wet with PRs on GitHub! Why not take a stab at it? It's pretty easy:

1. Fork the repo (in this case, [our CoreLocation repo](https://github.com/PromiseKit/CoreLocation))
2. Add the change (it'll go in the [CLGeocoder+Promise.swift file](https://github.com/PromiseKit/CoreLocation/blob/master/Sources/CLGeocoder%2BPromise.swift))
    - Bonus: Implement a test for it copying our current [CLGeocoder tests](https://github.com/PromiseKit/CoreLocation/blob/master/Tests/CLGeocoderTests.swift)
3. Open the PR!

**Comment by timbms** [comment]
OK I take the challenge.
Let me do it over the weekend.
Tim

On Tue, Feb 20, 2018 at 11:03 PM, Nathan Hosselton <notifications@github.com

**Comment by mxcl** [comment]
Closed by #12
